tests/nightly/classic-ubuntu-core-transition-auth/task.yaml

```diff
@@ -34,9 +47,12 @@ execute: |
     snap ack ./ubuntu-core_*.assert
     snap install ./ubuntu-core_*.snap
 
-    mkdir -p /root/.snap/
+    ensure_dir_exists_backup_real /root/.snap
+    ensure_file_exists_backup_real /root/.snap/auth.json
     echo '{}' > /root/.snap/auth.json
-    mkdir -p /home/test/.snap/
+
+    ensure_dir_exists_backup_real /home/test/.snap
```

Comment:
Does this create the file as the test user? If not then this is probably not a realistic scenario.

Reply:
Agree, test is previously created during the preparation of the suite.